.github/workflows/website-validation.yml

```diff
@@ -63,6 +66,34 @@ jobs:
           pnpm i --frozen-lockfile
           pnpm lint
 
+  # Run the website's tests.
+  test:
+    runs-on: ubuntu-latest
+    name: 🧪 Website Tests
+    needs: build_site
+    steps:
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # ratchet:actions/checkout@v3
```

Comment:
What does the ratchet command do? Does it shorten the action?

Reply:
[Ratchet](https://github.com/sethvargo/ratchet) is just the "overall tool". The step is really an `actions/checkout` of the repo.

Reply:
Looks like a tool for pinning dependency versions for consistency in GitHub actions.
https://github.com/sethvargo/ratchet